app/models/user.rb

```diff
@@ -507,13 +507,13 @@ def muted_user_ids
   def unread_notifications_of_type(notification_type)
     # perf critical, much more efficient than AR
     sql = <<~SQL
-        SELECT COUNT(*)
-          FROM notifications n
-     LEFT JOIN topics t ON t.id = n.topic_id
-         WHERE t.deleted_at IS NULL
-           AND n.notification_type = :notification_type
-           AND n.user_id = :user_id
-           AND NOT read
+         SELECT COUNT(*)
+           FROM notifications n
+      LEFT JOIN topics t ON t.id = n.topic_id
+          WHERE t.deleted_at IS NULL
+            AND n.notification_type = :notification_type
+            AND n.user_id = :user_id
+            AND NOT read
```

Comment:
Is there a reason we indented this by one extra space? The previous indentation comes to 2 spaces per index.

Reply:
Did that accidentally, for some reason my IDE decided it will be better this way :). I've reverted this reformatting.